src/full/Agda/TypeChecking/Constraints.hs

```diff
@@ -130,8 +130,9 @@ noConstraints problem = do
   (pid, x) <- newProblem problem
   cs <- getConstraintsForProblem pid
   unless (null cs) $ do
-    w <- warning_ (UnsolvedConstraints cs)
-    typeError $ NonFatalErrors [ w ]
+    withFileAndLine $ \ file line -> do
+      w <- warning'_ (file, line) (UnsolvedConstraints cs)
```

Comment:
Seems these two were forgotten...

Reply:
I would have sworn that `make type-check` went through but clearly that is
not the case. :disappointed: